azurerm/resource_arm_image.go

```diff
@@ -35,6 +35,13 @@ func resourceArmImage() *schema.Resource {
 
 			"resource_group_name": resourceGroupNameSchema(),
 
+			"zone_resilient": {
```

Comment:
could we also Set this in the state here using the object returned from the API? https://github.com/terraform-providers/terraform-provider-azurerm/pull/3100/files#diff-95b4abb1c5040a8fd4f3c3674821b7b6R292